my-travel-map/src/app/api/blog/post.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const sqlite3 = require('sqlite3');
+
+const app = express();
+app.use(express.json());
+
+// Provide the correct path to the existing myBlog.db
+const dbPath = '/Users/zhuminxuan/world-map-album-1/my-travel-map/myBlog.db'; // Replace with the actual path to your myBlog.db
+const db = new sqlite3.Database(dbPath, sqlite3.OPEN_READWRITE, (err) => {
```

Comment:
You should create something like [database.js](https://github.com/fredhu0514/world-map-album/blob/main/my-travel-map/src/database/database.js), instead of directly editing databse in your API file.